tests/common/runner.rs

```diff
@@ -20,11 +19,24 @@ pub fn mock_account(idx: usize) -> (Address, PlainAccount) {
     )
 }
 
-// Build an in memory storage for execution.
+// Build InMemoryStorage.
 pub fn build_in_mem(
     accounts: impl IntoIterator<Item = (Address, PlainAccount)>,
 ) -> InMemoryStorage {
-    accounts.into_iter().collect::<ChainState>().into()
+    build_in_mem_with_block_hashes(accounts, [])
+}
+
+// Build InMemoryStorage with block hashes.
+pub fn build_in_mem_with_block_hashes(
+    accounts: impl IntoIterator<Item = (Address, PlainAccount)>,
+    block_hashes: impl IntoIterator<Item = (U256, B256)>,
+) -> InMemoryStorage {
+    InMemoryStorage::new(
+        accounts
+            .into_iter()
+            .map(|(address, account)| (address, account.into())),
```

Comment:
This is for a future PR but the world will be a better place when we replace all `PlainAccount` with our `Account` (after renaming from `InMemoryAccount`). Then I think the only conversion needed is in the state test where we would convert our `Account` to `PlainAccount` to call their `state_merkle_trie_root` function. In a further future we'd implement that ourselves, too.

The end goal is to have consistent types both internally and for library users and minimize leaking underlying types like REVM's anywhere.

Reply:
ok, I will add a comment